gdpr/anonymizers.py

```diff
@@ -36,19 +38,22 @@ def get_anonymized_value_from_obj(self, obj, name):
             return self.get_anonymized_value(value)
 
     def get_anonymized_value(self, value):
-        """
-        There must be defined implementation of rule for anonymization
-        @param value: value that is anonymized
-        @return: anonymized value
+        """There must be defined implementation of rule for anonymization
```

Comment:
to there dej na nový řádek, tohle je blbě čitelné

Reply:
viz https://www.python.org/dev/peps/pep-0257/

Reply:
Upraveno.